src/source/Sctp/Sctp.c

```diff
@@ -135,11 +135,15 @@ STATUS freeSctpSession(PSctpSession* ppSctpSession)
 
     CHK(pSctpSession != NULL, retStatus);
 
+    usrsctp_deregister_address(pSctpSession);
```

Comment:
Please include some comments on what you are doing and how you model it through sctp public APIs